pyspeckit/spectrum/fitters.py

```diff
@@ -1448,7 +1416,7 @@ def moments(self, fittype=None, **kwargs):
             The registered fit type to use for moment computation
         """
         fittype = fittype or self.fittype
-        return self.Registry.singlefitters[fittype].moments(
+        return self.Registry.multifitters[fittype].moments(
```

Comment:
OK, I am revising that comment: what you've done is OK.  BUT, we need to modify peakbgfit now...